examples/Improved_Vis.ipynb

```diff
@@ -0,0 +1,156 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "\n",
+    "from IPython.display import JSON, display\n",
+    "\n",
+    "import ipycytoscape\n",
+    "import ipywidgets as W\n",
+    "import networkx as nx\n",
+    "import rdflib\n",
+    "from ipycytoscape import CytoscapeWidget, Edge, Node\n",
+    "from ipyradiant import (\n",
+    "    FileManager,\n",
+    "    GetOutgoingPredicateObjects,\n",
+    "    InteractiveViewer,\n",
+    "    PathLoader,\n",
+    ")\n",
+    "from ipyradiant.rdf2nx import RDF2NX, URItoID, URItoShortID\n",
+    "from rdflib.namespace import RDFS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up graph using base from `RDF_to_NX.ipynb`"
```

Comment:
I would add a basic explanation below this that we are populating the widget for the example, and that this would be handled by another widget.